src/shared/modules/badgeup-client/components/achievement-earned-popup/achievement-earned-popup.ts

```diff
@@ -17,4 +25,34 @@ export class AchievementEarnedPopupComponent {
         this.achievement = this.navParams.get('achievement');
         this.earnedAwards = this.navParams.get('earnedAwards');
     }
+
+    public async ngOnInit() {
+
+        const url = this.achievement.meta.icon || BASE_ICON;
```

Comment:
seems like we should be using the icon getter here

Reply:
yes! sorry, I was doing multiple things and forgot 😅